src/processor/map/map_copy_from.cpp

```diff
@@ -87,9 +87,23 @@ std::unique_ptr<PhysicalOperator> PlanMapper::mapCopyNodeFrom(LogicalOperator* l
         columnTypes.push_back(expr->getDataType().copy());
         columnEvaluators.push_back(exprMapper.getEvaluator(expr));
     }
-    auto info =
-        std::make_unique<NodeBatchInsertInfo>(nodeTableEntry, storageManager->compressionEnabled(),
-            std::move(columnTypes), std::move(columnEvaluators), copyFromInfo->columnEvaluateTypes);
+    bool ignoreErrors = CopyConstants::DEFAULT_IGNORE_ERRORS;
+    if (copyFromInfo->source->type == common::ScanSourceType::FILE) {
```

Comment:
I'm assuming `IGNORE_ERRORS` only applies when copying from a file right? I don't think we have options when copying from a subquery.

Reply:
Nope. We might need to add it later.